src/utils/message_builder.rs

```diff
@@ -723,6 +727,190 @@ impl fmt::Display for MessageBuilder {
     }
 }
 
+
+/// Content modifier enum
+///
+/// Provides an enum of formatting modifiers for a string, for combination with
+/// string types and Content types.
+///
+/// # Examples
+///
+/// Create a new Content type which describes a bold-italic "text":
+///
+/// ```rust
```

Comment:
This doesn't assert anything, so it doesn't need to actually run, it only needs to compile. This can be done by appending `,no_run`.